spec/models/collection_spec.rb

```diff
@@ -502,6 +502,104 @@
+      <link href="http://www-airs.jpl.nasa.gov/" hreflang="en-US" title="AIRS home page at NASA/JPL. General information on the AIRS instrument, algorithms, and other AIRS-related activities can be found. (VIEW PROJECT HOME PAGE)" rel="via"/>
+      <link href="http://disc.gsfc.nasa.gov/AIRS/index.shtml" hreflang="en-US" title="AIRS data support home page at Goddard Earth Sciences Data Information and Service Center/Distributed Active Archive Center (GES DISC DAAC). Online documentation, software to read, visualize and analyze AIRS data and data access information are available from this web site. (VIEW PROJECT HOME PAGE)" rel="via"/>
+      <link href="ftp://airsl1.gesdisc.eosdis.nasa.gov/ftp/data/s4pa//Aqua_AIRS_Level1/AIRVBRAD.005/doc/README.AIRVBRAD.pdf" hreflang="en-US" title="product README file (VIEW RELATED INFORMATION : USER'S GUIDE)" rel="describedBy" type="application/pdf"/>
+      <link href="http://gcmd.nasa.gov/getdif.htm?GES_DISC_AIRVBRAD_V005" hreflang="en-US" type="text/html" rel="enclosure" title="AIRVBRAD"/>
+      <link href="https://fedeo.esa.int/opensearch/description.xml?parentIdentifier=EOP:ESA:EARTH-ONLINE:ADAM.Surface.Reflectance.Database" hreflang="en-US" type="application/opensearchdescription+xml" rel="search" title="Non-CMR OpenSearch Provider Granule Open Search Descriptor Document"/>
+      <link href="#{ENV['public_catalog_rest_endpoint']}concepts/C190465571-GSFCS4PA.xml" hreflang="en-US" type="application/xml" rel="via" title="Product metadata"/>
+      <dc:identifier>C190465571-GSFCS4PA</dc:identifier>
+      <dc:date>2002-08-30T00:00:00.000Z/</dc:date>
+      <echo:datasetId>AIRS/Aqua Level 1B Visible/Near Infrared (VIS/NIR) geolocated and calibrated radiances V005</echo:datasetId>
+      <echo:shortName>AIRVBRAD</echo:shortName>
+      <echo:versionId>005</echo:versionId>
+      <echo:dataCenter>GSFCS4PA</echo:dataCenter>
+      <echo:archiveCenter>GEOSDISC</echo:archiveCenter>
+      <echo:tag>
+        <echo:tagKey>opensearch.granule.osdd</echo:tagKey>
+        <echo:data>"https://fedeo.esa.int/opensearch/description.xml?parentIdentifier=EOP:ESA:EARTH-ONLINE:ADAM.Surface.Reflectance.Database"</echo:data>
+      </echo:tag>
+      </entry>
+  </feed>
+      eos
+
+      catalog_rest_doc = Nokogiri::XML(cat_rest_response_str) do |config|
+        config.default_xml.noblanks
+      end
+      opensearch_doc = Nokogiri::XML(open_search_response_str) do |config|
+        config.default_xml.noblanks
+      end
+      actual = @d.to_open_search_collections(catalog_rest_doc, 1, {:clientId => "foo", :keyword => "AIRVBRAD", :cursor => "1", :numberOfResults => "10"}, "#{ENV['opensearch_url']}/datasets?cursor=1&numberOfResults=10")
+      expect(actual.to_xml(:indent => 2)).to eq(opensearch_doc.to_xml(:indent => 2))
+    end
+  end
+
```

Comment:
We have no existing tests that would break on the change you made?

Reply:
Correct. OSDD insertion for Non-CMR OpenSearch providers is tested, but only for the case where the provider had the additional `int.esa.fedeo` tag. This is why no tests broke when the extra `is_fedeo` check was originally added a few months ago - this new test covers that case.